src/main/java/xyz/nucleoid/plasmid/fake/FakeBlock.java

```diff
@@ -1,13 +1,15 @@
 package xyz.nucleoid.plasmid.fake;
 
+import eu.pb4.polymer.block.VirtualBlock;
 import net.minecraft.block.Block;
 import net.minecraft.block.BlockState;
 import net.minecraft.fluid.FluidState;
 
 /**
  * Represents a block that should be remapped to some vanilla "proxy" counterpart when being sent to clients
  */
-public interface FakeBlock {
+@Deprecated
```

Comment:
Can we add documentation to all these deprecated classes to rather point towards the correct replacement? 🙂

Reply:
Something like 
```
/**
 * @deprecated This interface is deprecated in favour of {@link eu.pb4.polymer.block.VirtualBlock}
 *
 * Represents a block that should be remapped to some vanilla "proxy" counterpart when being sent to clients
 */
``` should work?